src/go/rpk/pkg/api/admin/admin.go

```diff
@@ -245,6 +251,16 @@ func (a *AdminAPI) sendAny(method, path string, body, into interface{}) error {
 	return err
 }
 
+// General purpose backoff, includes 503s and other errors
+const retryBackoff = time.Duration(1500) * time.Millisecond
+
+// When there is no leader, we wait long enough for an election to complete
+const noLeaderBackoff = time.Duration(1500) * time.Millisecond
+
+// When there is a stale leader, we might have to wait long enough for
+// an election to start *and* for the resulting eleciton to complete
+const staleLeaderBackoff = time.Duration(9000) * time.Millisecond
+
```

Comment:
Please move these to the top of the file 🙏🏽

Reply:
Actually, since these constants are only used in the single function below, for maintenance purposes & scoping, can these be moved to the top of the function below in a single const block?

```go
func (a *AdminAPI) sendToLeader(
        method, path string, body, into interface{},
) error {
        const (
                // General purpose backoff, includes 503s and other errors.
                retryBackoff = 3 * time.Second / 2

                // When there is no leader, we wait long enough for an election
                // to complete.
                noLeaderBackoff = 3 * time.Second / 2

                // When there is a stale leader, we might have to wait long
                // enough for an election to start *and* for the resulting
                // election to complete.
                staleLeaderBackoff = 9 * time.Second
        )
```
(also fixed a small typo, `eleciton`, and switched to seconds but 1500ms may be clearer than 3*second/2)

Reply:
Moved these constants into the functions where they're used.